poetry.lock

```diff
@@ -54,6 +54,12 @@ optional = false
 python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*"
 version = "19.3.0"
 
+[package.extras]
```

Comment:
I think/hope that `poetry` now renders the `package.extras` list for each package directly into the lockfile, hence these few additional entries.  Would be nice to have a changelog reference for that though.

Reply:
Not totally certain (would have to do a bit of testing against previous versions), but it looks like this was introduced in https://github.com/python-poetry/poetry/pull/1205/commits/c29174809a0570d32686fc5689b0cd93797747da#diff-8f7a4c4761eabce6d17bef6ca760e2d7R211-R212, where the `locker` code now includes extras if present.  That was rolled into the first few `1.0.0` release candidates, and into the eventual release version.

Reply:
Ok, yep - this definitely changed between `poetry==0.12.17` and `poetry==1.0.0a0`, and in particular it was indeed commit c29174809a0570d32686fc5689b0cd93797747da which introduced the change.  From that point on, `extras` are rendered into the lockfile and this is definitely intended.